test_schema.py

```diff
@@ -240,6 +240,22 @@ def test_dict_subtypes():
     # is dropped!
 
 
+def test_dict_key_error():
+    try:
+        Schema({'k': int}).validate({'k': 'x'})
+    except SchemaError as e:
+        assert e.code == "Key 'k' error:\n'x' should be instance of 'int'"
+    try:
+        Schema({'k': {'k2': int}}).validate({'k': {'k2': 'x'}})
+    except SchemaError as e:
+        code = "Key 'k' error:\nKey 'k2' error:\n'x' should be instance of 'int'"
```

Comment:
Yes. It was longer than 90 before, so I stored the error message to `code` variable.
So I think my latest commit resolves PEP8